### ISSUE TITLE
Fetch tables does not remove deleted column - PostgreSQL

### ISSUE THREAD

**Issue by melohagan** [Budibase/budibase#6737]
**Hosting**
Self
  - Method: docker compose
  - Budibase Version: 1.1.4
  - App Version: 1.1.4

**Describe the bug**
If I delete a column from PostgreSQL in pgadmin, and refetch tables, Budibase does not remove the deleted column.
Discussion: https://github.com/Budibase/budibase/discussions/6586

**To Reproduce**
1. Fetch a PostgreSQL table
2. Within **pgadmin** delete a column from the table
3. You'll now see an error in the data table view (and no rows will load)
4. Refetching does not help

**Expected behavior**
Deleting a column from a PostgreSQL table outside of Budibase should not break your Budibase builder.

**Screenshots**
![Screenshot 2022-07-15 at 14 52 43](https://user-images.githubusercontent.com/101575380/179237097-78fed648-90da-4985-86e3-a81d68b88328.png)

**Comment by insomniacj93** [comment]
@melohagan Not only in PostgresSQL.
I'm using MySQL/docker and I'm Having the same issue, after noticing that my newly created tables weren't showing (but the changes to the existing tables did show) I decided to try and fetch the tables again. Now I've got my newly created tables, but some columns that no longer exist in MySQL still appear in budibase. On delete the action says it's done, yet the columns won't get removed.
